### PR TITLE
Billy Issue Fixes

### PR DESCRIPTION
**Pull request by WRRicht3r**
This PR also adds a Polkaswap SORA staking tutorial.
This PR addresses Issue #223 #230 #243 #234 #231 #232 #233 #235 #236 #237 #239 #240 and #241

### CODE REVIEW

File: src/snippets/explore-polkaswap.md [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
 
 ![](/.gitbook/assets/polkaswap-explore-staking.png)
 
-- [Tokens:](https://polkaswap.io/#/explore/tokens) The Tokens tab displays a list of all available tokens on the Polkaswap platform and provides detailed information about each token, including its current price, total value, token velocity, and trading volume. Users can filter the list by different criteria such as name, symbol, and trading volume.
+- [Tokens:](https://polkaswap.io/#/explore/tokens) The Tokens tab displays a list of all available tokens on the Polkaswap platform and provides detailed information about each token, including its current price, total value, token velocity, and trading volume. Users can filter the list by different criteria such as name, symbol, and trading volume, you can also toggle viewing only synthetic (XST) assets.
 
 ![](/.gitbook/assets/polkaswap-explore-tokens.png)
+
+- [Trade:](https://polkaswap.io/#/explore/books) The trade tab displays a list of the available orderbook pairs, including information such as the price, daily percentage and volume, and the total value locked. (*This diagram will be updated as more Orderbook pairs are added*.)
```
Comment (outoftardis) [inline src/snippets/explore-polkaswap.md]:
```suggestion
- [Trade:](https://polkaswap.io/#/explore/books) The trade tab displays a list of the available orderbook pairs, including information such as the price, daily percentage and volume, and the total value locked.
```

File: src/check-balance.md [hunk 2]
```diff
@@ -47,4 +56,22 @@ The letter before XOR means that the number is multiplied by the following amoun
 
 So, in the example above the shown amount is 80.434 mXOR = 0.080434 VAL
 
-For more details on how to read this output, [take a look at this](check-supply.md#how-to-read-the-results).
+For more details on how to read this output, [take a look at
+this](check-supply.md#how-to-read-the-results).
```
Comment (outoftardis) [inline src/check-balance.md]:
```suggestion
this guide](check-supply.md#how-to-read-the-results).
```

File: src/snippets/demeter-staking-polkaswap.md [hunk 3]
```diff
@@ -19,4 +19,132 @@
...
+	 
+ ![](/.gitbook/assets/polkaswap-staking-disclaimer.png)
+
+ ![](/.gitbook/assets/polkaswap-staking-validator-review.png)
+ 
+ 6. Confirm all the information is correct and sign the transaction. 
+ 
+ ![](/.gitbook/assets/polkaswap-staking-confirm.png)
+ 
+ 7. Alternatively, if you choose your on validators, from step 4,
```
Comment (outoftardis) [inline src/snippets/demeter-staking-polkaswap.md]: "your on validators" is it an option you can choose or is there a mistake/typo?

File: src/snippets/demeter-staking-polkaswap.md [hunk 4]
```diff
@@ -19,4 +19,132 @@
...
+   
+  :::info
+  If you have Validators you know and trust, you can choose them
+  manually, otherwise allow the algorithm to select them for you. You
+  can change your Validators any time you want.
+  :::
+  
+  ![](/.gitbook/assets/polkaswap-staking-choose-validators.png)
+  
+  5. If you choose the suggested Validators, you will get a disclaimer
```
Comment (outoftardis) [inline src/snippets/demeter-staking-polkaswap.md]: please make sure the items on the list are aligned: 1-4 are without indentation, 5-7 and 9 have indentation, 8 has different indentation

please fix this

File: src/snippets/demeter-staking-polkaswap.md [hunk 5]
```diff
@@ -19,4 +19,132 @@
...
+  
+  9. Confirm your selection then sign the transaction. You are now
+     nominating a SORA Validator. 
+  
+  ![](/.gitbook/assets/polkaswap-staking-confirm-own.png)
+
+### Reviewing your SORA Stake
+
+The next time you navigate to the Staking tab on Polkaswap, the SORA
+staking section will appear as as active and will display your APR.
```
Comment (outoftardis) [inline src/snippets/demeter-staking-polkaswap.md]:
```suggestion
staking section will appear as active and will display your APR. 
```

File: src/snippets/demeter-staking-polkaswap.md [hunk 6]
```diff
@@ -19,4 +19,132 @@
...
+
+### Reviewing your SORA Stake
+
+The next time you navigate to the Staking tab on Polkaswap, the SORA
+staking section will appear as as active and will display your APR. 
+
+ ![](/.gitbook/assets/polkaswap-staking-active.png)
+
+When you click on the SORA Staking section, you will be able to review
+the statistics of your stake, as well as add and remove stake from
```
Comment (outoftardis) [inline src/snippets/demeter-staking-polkaswap.md]:
```suggestion
the statistics of your stake, as well as add and remove the stake from
```

File: src/snippets/demeter-staking-polkaswap.md [hunk 7]
```diff
@@ -19,4 +19,132 @@
...
+
+ ![](/.gitbook/assets/polkaswap-staking-more.png)
+ 
+ To remove some or all of your stake of XOR, click on Remove Stake,
+ then select the percentage of your stake to remove. 
+ 
+ :::info
+ The unstaking period is 7 days from when you confirm the transaction
+ to remove your stake.
+ :::
```
Comment (outoftardis) [inline src/snippets/demeter-staking-polkaswap.md]:
```suggestion
To remove some or all of your stake of XOR, click on Remove Stake,
then select the percentage of your stake to remove. 
 
:::info
The unstaking period is 7 days from when you confirm the transaction
to remove your stake.
:::
```

File: src/snippets/demeter-staking-polkaswap.md [hunk 8]
```diff
@@ -19,4 +19,132 @@
...
+   ![](/.gitbook/assets/polkaswap-staking-validators-all.png) 
+   
+   To edit the Validators you choose to nominate, from the "Your
+   Validator" view, click "Change Validators."
+   
+   ![](/.gitbook/assets/polkaswap-staking-change-validators.png) 
+   
+   You will be given the option once again to select suggested
+   Validators or pick them by yourself. Pick whichever you prefer then
+   confirm the changes by signing the transaction.
```
Comment (outoftardis) [inline src/snippets/demeter-staking-polkaswap.md]: please remove indentation

File: src/snippets/demeter-staking-polkaswap.md (hunk 3 above)
Comment (WRRicht3r) [reply src/snippets/demeter-staking-polkaswap.md]: its a typo, should be "your own"

Comment (WRRicht3r) [reply src/snippets/demeter-staking-polkaswap.md]: Fixed

File: src/snippets/demeter-staking-polkaswap.md (hunk 4 above)
Comment (WRRicht3r) [reply src/snippets/demeter-staking-polkaswap.md]: Fixed

File: src/snippets/demeter-staking-polkaswap.md (hunk 8 above)
Comment (WRRicht3r) [reply src/snippets/demeter-staking-polkaswap.md]: Fixed

**Review by outoftardis — APPROVED** [review]

**Review by bgodlin — APPROVED** [review]

**Pull request merged**